content/foundations/color/overview.mdx

```diff
@@ -0,0 +1,148 @@
+
+<img
+  width="960"
+  alt="wip"
+  src="https://github.com/primer/react/assets/18661030/3e4ec1f8-cbc5-4ebf-9816-7f247474b9c6"
+/>
+
+### Accent foreground
+
+Foreground accent colors provide contrast against `muted` and default background colors and should be used for text and icons.
+
+<img
+  width="960"
+  alt="wip"
+  src="https://github.com/primer/react/assets/18661030/7fcd6efb-ec86-4460-98aa-e3e5be062bec"
+/>
+
+## Color roles
+
+| Color roles                                                                       | Usage                                                               |
+| :-------------------------------------------------------------------------------- | :------------------------------------------------------------------ |
+| <Box sx={{display: 'inline-flex', alignContent: 'center', gap: '4px', verticalAlign: 'middle'}}><Box sx={{backgroundColor: 'accent.emphasis', height: '24px', width: '24px', borderRadius: 1}}></Box> `accent`</Box> | Links, selected, active, and focus states, and neutral information |
+| <Box sx={{display: 'inline-flex', alignContent: 'center', gap: '4px', verticalAlign: 'middle'}}><Box sx={{backgroundColor: 'success.emphasis', height: '24px', width: '24px', borderRadius: 1}}></Box> `success`</Box>     | Primary buttons, positive messaging and successful states                                |
+| <Box sx={{display: 'inline-flex', alignContent: 'center', gap: '4px', verticalAlign: 'middle'}}><Box sx={{backgroundColor: 'attention.emphasis', height: '24px', width: '24px', borderRadius: 1}}></Box> `attention`</Box> | Warning states, active processes such as queued PRs and tests in progress   |
+| <Box sx={{display: 'inline-flex', alignContent: 'center', gap: '4px', verticalAlign: 'middle'}}><Box sx={{backgroundColor: 'danger.emphasis', height: '24px', width: '24px', borderRadius: 1}}></Box> `danger`</Box>       | Danger buttons and error states |
+| <Box sx={{display: 'inline-flex', alignContent: 'center', gap: '4px', verticalAlign: 'middle'}}><Box sx={{backgroundColor: 'open.emphasis', height: '24px', width: '24px', borderRadius: 1}}></Box> `open`</Box>           | Open tasks, PRs or workflows           |
+| <Box sx={{display: 'inline-flex', alignContent: 'center', gap: '4px', verticalAlign: 'middle'}}><Box sx={{backgroundColor: 'closed.emphasis', height: '24px', width: '24px', borderRadius: 1}}></Box> `closed`</Box>       | Closed tasks, PRs or workflows         |
+| <Box sx={{display: 'inline-flex', alignContent: 'center', gap: '4px', verticalAlign: 'middle'}}><Box sx={{backgroundColor: 'done.emphasis', height: '24px', width: '24px', borderRadius: 1}}></Box> `done`</Box>           | Completed tasks, PRs or workflows      |
+| <Box sx={{display: 'inline-flex', alignContent: 'center', gap: '4px', verticalAlign: 'middle'}}><Box sx={{backgroundColor: 'sponsors.emphasis', height: '24px', width: '24px', borderRadius: 1}}></Box> `sponsors`</Box>           | Text and icons related to GitHub Sponsors      |
```

Comment:
Below this could also be an appropriate place to link to the color theme reference tables `/foundations/primitives/color`, if users are devs scrolling to find color values.

Reply:
Awesome!

<img width="895" alt="section for devs" src="https://github.com/primer/design/assets/18661030/573d4d57-7127-4137-ac46-40b3c5d555a7">